### ISSUE TITLE
Add some sort of pagination

### ISSUE THREAD

**Issue by Buccaneer** [goegefietst/server#8]
As the amount of routes will increase we should add some sort of pagination to query for routes

**Comment by pietercolpaert** [comment]
Is this implemented?

**Comment by Buccaneer** [comment]
No, there's currently only one way to query for routes which is by user id and gives all routes for that user id. You also need to know the user secret to do this.
